### ISSUE TITLE
Improve spam prevention measure on form submissions

### ISSUE THREAD

**Issue by srtalbot** [cds-snc/platform-forms-client#409]
**User story**: Persona based
As an end-user using any interaction device (mouse, screen, screen reader, other adaptive technology)
I need to know what my next steps are if I am suspected as a bot by reCAPTCHA V3, or if I am denied service
So that I can follow up via a different channel and continue to get the service I need

As a program administrator processing form responses
I need to focus my time on legitimate responses
So that I can decrease the processing time of requests

As a program administrator
I need to spend my time processing legitimate forms
So that my program can meet operational deadlines

**Threat level:**
Spam prevention level 2: We can stop a malicious actor who is writing a custom script to spam a single (or multiple) GC Form(s). 

ITSG-33 controls: SC-5, SI-10

**Story context**
*Newest documentation is first in list.* 
[Service flow for spam prevention](https://miro.com/app/board/uXjVOPZ1Eps=/) using recaptcha V3
[Options analysis for spam prevention](https://docs.google.com/document/d/1AIs_mb4Mu4HFXJ_vSBah-ONnxhiuEG4t9gKg7VDxZpE/edit#)

[Risk register entry](https://docs.google.com/spreadsheets/d/1AyDSS2pnGtHtEow4eXx8tRwmExK9lrQJ6WqRAZ0ElEA/edit#gid=0&range=21:21)
[Thread on spam prevention ideas from Pat Heard](https://gcdigital.slack.com/archives/C01GFD150MC/p1631192319040300)
[WCAG research on captcha](https://www.w3.org/TR/turingtest/)
[Previous discussion in #interaction-design, with recommendation from Julianna](https://gcdigital.slack.com/archives/CNLK8G26P/p1615233098002300)
[Recommended option for analysis](https://www.google.com/recaptcha/about/)

**Design Documentation**
- [X] [Options analysis with trade-offs for captcha implementations](https://docs.google.com/document/d/1XA4Qcsm05FfvSe3H_g68mOXj88OrwJQB_oW44HY8GqU/edit#heading=h.rguwvkowez10)
- [x] Are there privacy concerns with using Captcha v2, v3, or enterprise?
- [x] What does an unhappy path look like if a legitimate user fails the test. 
- [x] Content surrounding the captcha (context for captcha use and fail paths)
- [x] Identify spam prevention options (starting and ideal)
- [x] Review terms and conditions language for GC Forms

**Technical implementation**
- [ ] Define implementation approach
- [ ] Create wireframes for additional screens and service flow
- [ ] Draft content for happy and unhappy paths
- [ ] Review the minimum time requirement on form submissions
- [ ] QA test the minimum time requirement functionality
- [ ] Turn feature flag on

**Comment by srtalbot** [comment]
Identify and implement spam prevention measures

Acceptance criteria:
- [ ] review IP rate limit cut-off
- [ ] review decision to implement a captcha
- [ ] ... _input required_

**Comment by hillaryl** [comment]
A review of captcha accessibility: https://www.w3.org/TR/2019/NOTE-turingtest-20191209/

Seems like Google reCAPTCHA v2 is the most accessible options, but still has some accessibility problems along with privacy concerns that may not work for us.

**Comment by timarney** [comment]
Will close this as stale --- Hcaptcha work is on staging.
